include/eve/module/real/core/function/regular/generic/load.hpp

```diff
@@ -43,7 +43,67 @@ namespace eve::detail
   constexpr bool sanitizers_are_on = false;
 #endif
 
+  //================================================================================================
+  // Bundle load
+  //================================================================================================
+  template<typename... Pointers>
+  EVE_FORCEINLINE auto load_( EVE_SUPPORTS(cpu_), kumi::tuple<Pointers...> ptr) noexcept
+  {
+    auto card = expected_cardinal_t< kumi::tuple<std::remove_cvref_t<decltype(*std::declval<Pointers>())>...> >{};
+    return eve::load(ptr, card);
+  }
+
+/*  template<typename... Pointers,typename Cardinal>
+  EVE_FORCEINLINE auto load_( EVE_SUPPORTS(cpu_), kumi::tuple<Pointers...> ptr
+                            , Cardinal
+                            ) noexcept
+  {
+    return bundle { kumi::map ( [](auto p) { return eve::load(p, Cardinal{}); }, ptr) };
+  }
+*/
```

Comment:
Delete?

Reply:
it's a placeholder for me, code will appear here soon